### PR TITLE
Added 'Layer5 Team' author option for blogs

### LINKED ISSUE #2333: [Blogs] Add "Layer5 Team" author type
**Current Behavior**
Layer5 blog posts are written by any number of representatives of the team, often done collaboratively.

**Desired Behavior**
A team byline is needed (see mockup). When the `author` frontmatter field matches "Layer5 Team" the following logo and "Layer5 Team" should be shown in single blog post views.

---
**Screenshots / Mockups**
Screen Shot 2021-11-09 at 7 54 35 AM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
**Description**
Updated the code to use the layer image for the blogs which have the authors as "Layer5 Team".

This PR fixes #2333 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
@leecalcote for the blogs which had "Layer5 Contributor" as the author I have changed them to "Layer5 Team", is that fine?

**Comment by l5io** [comment]
🚀 Preview for commit c0265a6fe225a5910e794c4845bcb9144de14be3 at: https://618a9b9b1cfd4000a08e28ee--layer5ng.netlify.app

**Comment by leecalcote** [comment]
> @leecalcote for the blogs which had "Layer5 Contributor" as the author I have changed them to "Layer5 Team", is that fine?

Yes, sir. That is perfect. 💯

File: src/reusecore/PageHeader/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import {Link, graphql, useStaticQuery} from "gatsby";
 import slugify from "../../utils/slugify";
 import PageHeaderWrapper from "./pageHeader.style";
 import Image from "../../components/image";
+import layer5_img from "../../assets/images/layer5/layer5-only/svg/layer5-gray-no-trim.svg";
```
Comment (leecalcote) [inline src/reusecore/PageHeader/index.js:7]: Wow. I don't know whether to be impressed or to just expect this from 2020's MeshMate of the Year. With great precision, you selected the extact flavor of this logo - the svg, the gray, the no trim, the no tagline... 🎯

Comment (Nikhil-Ladha) [reply src/reusecore/PageHeader/index.js:7]: Haha, it's been some time doing neat work 😎 like this!

**Review by leecalcote — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
For ref: https://618a9b9b1cfd4000a08e28ee--layer5ng.netlify.app/blog/performance/performance-benchmarking-using-meshery-and-nighthawk an affected blog post.

**Comment by Nikhil-Ladha** [comment]
Merging..... 🚀

**Comment by leecalcote** [comment]
Screen Shot 2021-11-09 at 10 40 06 AM

A MeshMate move: an italicized skew added to the Layer5 logo.

**Pull request merged**